backend/app/models/segments.py

```
@@ -0,0 +1,41 @@
+    __tablename__ = "content_segments"
+
+    id: uuid.UUID = Field(default_factory=uuid.uuid4, primary_key=True, index=True)
+
+    # 优化查询性能的索引
+    __table_args__ = (
+        Index('idx_segments_item_number', 'content_item_id', 'display_number'),
+        Index('idx_segments_item_created', 'content_item_id', 'created_at'),
+    )
+
+    # 关联关系
+    content_item: "ContentItem" = Relationship(
```

Ruff (0.12.2)

Undefined name `ContentItem`

(F821)